src/main.ts

```diff
@@ -21,7 +21,7 @@ export async function run(): Promise<void> {
 
     // get audit-level
     const auditLevel = core.getInput('audit_level', {required: true})
-    if (!['critical', 'high', 'moderate', 'low'].includes(auditLevel)) {
+    if (!['critical', 'high', 'moderate', 'low', 'none'].includes(auditLevel)) {
```

Comment:
```suggestion
    if (!['critical', 'high', 'moderate', 'low', 'info', 'none'].includes(auditLevel)) {
```
The documentation you told me also includes "info".
https://docs.npmjs.com/cli/v8/commands/npm-audit#audit-level